utils.js

```diff
@@ -217,20 +217,42 @@ export function getLicenses(pkg, format = "xml") {
             licenseContent.id = l;
             licenseContent.url = "https://opensource.org/licenses/" + l;
           } else if (l.startsWith("http")) {
-            if (!l.includes("opensource.org")) {
-              licenseContent.name = "CUSTOM";
-            } else {
+            if (
+              l.includes("mit-license") ||
+              l.includes("://github.com/dotnet/standard/") ||
+              l.includes("://github.com/dotnet/corefx/") ||
```

Comment:
Any ideas how you identified these urls as mit? Also any chance we can use the file https://github.com/CycloneDX/cdxgen/blob/master/data/known-licenses.json for corrections?

Reply:
Thanks for the hint, way better. Will take care next week.

Reply:
About identifying as MIT. I check several releases of the libraries:

- https://github.com/dotnet/standard/
  - https://github.com/dotnet/standard/blob/release/3.0/LICENSE.TXT
  - https://github.com/dotnet/standard/blob/release/2.0.0/LICENSE.TXT

- https://github.com/dotnet/corefx/
  - https://github.com/dotnet/corefx/blob/release/1.0.0/LICENSE
  - https://github.com/dotnet/corefx/blob/release/1.1.0/LICENSE
  - https://github.com/dotnet/corefx/blob/release/2.0.0/LICENSE.TXT
  - https://github.com/dotnet/corefx/blob/release/3.1-blazor/LICENSE.TXT